MariiaHuk/HW5/mh5.1.py

```diff
@@ -0,0 +1,4 @@
+integer_list = [1, 5, 9, 17, 20]
```

Comment:
Your previous code is simple and functional. However, a more Pythonic approach can make it concise. Here's a revised version:
integer_list = [1, 5, 9, 17, 20]

# Using list comprehension to convert each element to float
float_list = [float(num) for num in integer_list]

# Printing the resulting list
print(float_list)